### PR TITLE
Added LICENSE and modernized gemspec

### PR DESCRIPTION
**Pull request by smathieu**

### CODE REVIEW

File: lib/queue_classic/version.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module QC
+  VERSION = "3.1.0RC"
```
Comment (senny) [inline lib/queue_classic/version.rb]: there is a `3-1-stable` branch already. I think `master` should be version `3.2.0.alpha`

**Comment by senny** [comment]
@smathieu can we name the file just `LICENSE` without the `.txt`?

**Comment by smathieu** [comment]
I don't feel very strong in favour of `LICENSE` vs `LICENSE.txt`, but I named it .txt because when you run `bundle gem ...` it creates a `LICENSE.txt` file.

File: lib/queue_classic/version.rb (hunk 1 above)
Comment (smathieu) [inline lib/queue_classic/version.rb]: Thanks, I wasn't sure which to use. Will update.

**Comment by senny** [comment]
@smathieu let's trust what bundler does then. `LICENSE.txt` it is :grin:

**Comment by jipiboily** [comment]
:heart:

**Pull request merged**